src/js/components/Skeleton/index.d.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import {
+  A11yTitleType,
+  AlignSelfType,
+  GridAreaType,
+  MarginType,
+  HeightType,
+  PadType,
+  PolymorphicType,
+  RoundType,
+  SkeletonColorsType,
+  WidthType,
+} from '../../utils';
+
+
+export interface SkeletonProps {
+  a11yTitle?: A11yTitleType;
+  alignSelf?: AlignSelfType;
+  as?: PolymorphicType;
+  colors?: SkeletonColorsType;
+  gridArea?: GridAreaType;
+  margin?: MarginType;
```

Comment:
do we need `message` prop?

Reply:
I didn't think so since the intent was the main box container which has the context was owning that. I'm less clear about the direct usage of the Skeleton component since it seemed more valuable to just do it via a Box with the skeleton prop and not generally use Skeleton directly. Curious to see how the use model evolves

Reply:
maybe we can discuss ☝️